packages/token-bridge-sdk/src/test/graph.test.ts

```diff
@@ -0,0 +1,60 @@
+import 'cross-fetch/polyfill'
```

Comment:
should we move this to `packages/token-bridge-sdk/test/graph.test.ts`?
as is I think its getting included in the dist